src/views/Package/Package.tsx

```diff
@@ -18,6 +19,36 @@ interface PathParams {
   version: string;
 }
 
+const docsOrError = (
+  markdown: UseRequestResponse<string>,
+  assembly: UseRequestResponse<spec.Assembly>,
+  language: string
+) => {
+  if (markdown.error || assembly.error) {
+    return (
+      <Text
+        align="center"
+        fontSize="xl"
+        fontStyle="oblique"
+        wordBreak="break-word"
+      >
+        Oops, Looks like {language} documentation is not available yet. Checkout
+        other languages in the meantime!
+      </Text>
+    );
+  }
+
+  if (markdown.loading || assembly.loading) {
+    return null;
+  }
+
+  if (!markdown.data || !assembly.data) {
```

Comment:
Imo we can just consolidate these into one if statement

Reply:
👍